hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/shell/TestOzoneShellHA.java

```diff
@@ -558,6 +560,86 @@ public void testDeleteToTrashOrSkipTrash() throws Exception {
     }
   }
 
+  @Test
+  public void testDeleteTrashNoSkipTrash() throws Exception {
+
+    // Test delete from Trash directory removes item from filesystem
+
+    // setup configuration to use TrashPolicyOzone
+    // (default is TrashPolicyDefault)
```

Comment:
Please use `getClientConfForOFS(hostPrefix,conf)` for initialising `clientConf` here. The integration test is failing because of this
```
 // Test delete from Trash directory removes item from filesystem

    // setup configuration to use TrashPolicyOzone
    // (default is TrashPolicyDefault)
    final String hostPrefix = OZONE_OFS_URI_SCHEME + "://" + omServiceId;
    OzoneConfiguration clientConf = getClientConfForOFS(hostPrefix,conf);
    clientConf.setInt(FS_TRASH_INTERVAL_KEY, 60);
    clientConf.setClass("fs.trash.classname", TrashPolicyOzone.class,
                        TrashPolicy.class);
    OzoneFsShell shell = new OzoneFsShell(clientConf);
```

Reply:
Thanks for the comment @sadanand48.  Commit to set configuration in function setting TrashPolicyOzone after getClientConfForOFS as suggested.

Reply:
Thanks @neils-dev for making the change . Sorry , I wasn't clear . I meant to use the same URI as that of `testDeleteToTrashOrSkipTrash` i.e to include serviceId as part of the URI.  Since this is HA setup ,its required to pass the serviceID.  Please use the same URI as that of `testDeleteToTrashOrSkipTrash`  i.e
```
final String hostPrefix = OZONE_OFS_URI_SCHEME + "://" + omServiceId;
OzoneConfiguration clientConf = getClientConfForOFS(hostPrefix,conf);

```
instead of 
`OzoneConfiguration clientConf =
            getClientConfForOzoneTrashPolicy("ofs://localhost", conf);`

Reply:
Thanks for your comment **@sadanand48** .  Commit a change for the service id as suggested.  Thanks for following up.  BTW, the host _set to_ `localhost`, as was previously used for the clientconf _should not_ be a problem with the integration test.  It was tested both individually` hadoop-ozone/integration-test$ mvn -Dtest=TestOzoneShellHA#testDeleteTrashNoSkipTrash test` and when the entire suite is run, `hadoop-ozone/integration-test$ mvn -Dtest=TestOzoneShellHA test`.  See attached image.

` hadoop-ozone/integration-test$ mvn -Dtest=TestOzoneShellHA#testDeleteTrashNoSkipTrash test`
![skipTrash](https://user-images.githubusercontent.com/81126310/114753821-24128680-9d15-11eb-81e5-c47b9bc75eb6.png)